src/components/Onboarding/OnboardingPage1.tsx

```diff
@@ -5,30 +5,30 @@ import * as Progress from 'react-native-progress';
 import { KeyboardAwareScrollView } from 'react-native-keyboard-aware-scroll-view';
 import DropDownPicker from 'react-native-dropdown-picker';
 import styles from '../../styles/OnboardingStyles';
-import states from './states';
+import { states, homeInfo } from './onboardingData';
 
 interface Props {
   setPage(page: string): void;
   setHomeData({}): void;
-  homeData: Record<string, string>;
+  homeData: typeof homeInfo;
```

Comment:
`homeData` can just be type `Home` from `API.ts`.

Reply:
I've described why homeInfo exists below.